wyvern/experimentation/experimentation_logging.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+from datetime import datetime
+
+from pydantic import BaseModel
+
+from wyvern.components.events.events import EventType, LoggedEvent
+
+
+class ExperimentationEventData(BaseModel):
+    experiment_id: str
+    entity_id: str
+    result: str
+    timestamp: datetime
+    metadata: dict
```

Comment:
```suggestion
    metadata: Dict
```
we're still supporting python 3.8 at this point.   `dict` won't work for python3.8